src/main/java/io/debezium/connector/yugabytedb/YugabyteDBStreamingChangeEventSource.java

```diff
@@ -272,11 +272,6 @@ private void getChanges2(ChangeEventSourceContext context,
             schemaNeeded.put(entry.getValue(), Boolean.TRUE);
         }
 
-        for (Pair<String, String> entry : tabletPairList) {
-            final String tabletId = entry.getValue();
-            offsetContext.initSourceInfo(tabletId, this.connectorConfig);
-        }
-
```

Comment:
Removed this block since it is overriding what was being initialized in the previous block.

Ideally this should not cause any issues since we call GetChanges from 0,0 and get from certain OpId and then update the offset.